portfolio/src/main/webapp/script.js

```diff
@@ -99,13 +101,25 @@ function createIndividualComment(comment) {
 function addUploadedImageToComment(comment) {
   const imageElement = document.createElement('img');
   imageElement.setAttribute('class', 'comment-image');
-  imageElement.src = comment.imageURL;
+  imageElement.src = `/serve?blob-key=${comment.blobKeyString}`;
   const imageAnchor = document.createElement('a');
-  imageAnchor.href = comment.imageURL;
+  imageAnchor.href = `/serve?blob-key=${comment.blobKeyString}`;
   imageAnchor.appendChild(imageElement);
   return imageAnchor;
 }
 
+function addImageLabels(labels) {
```

Comment:
Please add comments on what these functions actually do. Try to get into the habit of adding a short comment for every function/class/method.

Reply:
Done